app/models/invenio_rdm_record_converter.rb

```diff
@@ -611,4 +611,14 @@ def publisher
   def pediatric_neurology_brief?
     @generic_file.depositor == INSTITUTIONAL_PNB_DEPOSITOR || @generic_file.doi.shift.include?(PNB_DOI_PREFIX)
   end
+
+  def sizes
+    if @generic_file.id == "3105875f-61e1-412c-9b1c-c8a33b37ff35"
+      ["116 pages"]
+    elsif  @generic_file.id == "9cd9e3df-458c-4a2c-9e42-1dcdc95e7adf"
+      ["44 pages"]
+    elsif @generic_file.page_count
```

Comment:
You need to check if this is blank, right?

Reply:
Nope, we want to check if there is anything to format. If it's blank we will just skip it.

Reply:
Whoops, I meant not blank. Empty arrays are true, right?

Reply:
oh right! that was the reason it was using that logic